ci/run.yml

```diff
@@ -0,0 +1,30 @@
+- hosts: controller
+  tasks:
+    - name: Install ansible so that ansible-playbook is available
+      ansible.builtin.pip:
+        name:
+          - ansible
+
+    - name: Set the repo dir
+      ansible.builtin.set_fact:
+        fvt_dir: "{{ ansible_env.HOME}}/{{ zuul.projects['github.com/infrawatch/feature-verification-tests'].src_dir }}"
+
+    - name: Copy the callback_plugin to enable it
+      ansible.builtin.copy:
+        src: "{{ fvt_dir }}/callback_plugins/"
+        dest: "{{ fvt_dir }}/ci/callback_plugins/"
+
+    - name: "Run a playbook locally on the target host"
+      ansible.builtin.shell:
+        cmd: |
+          ANSIBLE_CONFIG=ansible.cfg ansible-playbook stf_tests.yml
+        chdir: "{{ fvt_dir }}/ci"
```

Comment:
I think that if you change this to ``chdir: "{{ fvt_dir }}"``, and `... ansible-playbook ci/stf_tests.yml`, that you don't need to move the callback_plugin dir.

Alternatively, you can tell ansible where to look for callback plugins, the same way that you tell ansible where to look for roles.
https://docs.ansible.com/ansible/latest/reference_appendices/config.html#default-callback-plugin-path

Reply:
fo some reason ts not working as For ansible to use the callback_plugins directory it must be located beside the playbook, in the same directory. we need to have something like that, I'm playing around to find a way.

├── callback_plugins
│   └── env_warning.py
└── playbook.yml